connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -184,6 +184,17 @@ public void resumeConnector(String connector) {
         configBackingStore.putTargetState(connector, TargetState.STARTED);
     }
 
+    @Override
+    public Map<String, String> maskCredentials(String connName, Map<String, String> config) {
+        Map<String, String> newConfig = new LinkedHashMap<>();
+        for (Map.Entry<String, String> entry : config.entrySet()) {
+            // Password.toString() will return the hidden value
+            newConfig.put(entry.getKey(), entry.getValue().toString());
```

Comment:
Can't the value of an entry be null?

Reply:
If the value of the entry is null; wouldn't `.toString()` print out "null"? 
https://stackoverflow.com/questions/29513497/why-tostring-on-an-object-instance-which-is-null-is-not-throwing-npe